Source/WebCore/Modules/modern-media-controls/controls/inline-media-controls.js

```diff
@@ -208,8 +208,11 @@ class InlineMediaControls extends MediaControls
         this.leftContainer.layout();
         this.rightContainer.layout();
 
+        // Update the list of shown buttons in the right container since it may have changed again.
+        shownRightContainerButtons = this.rightContainer.children.filter(button => button.enabled && !button.dropped);
```

Comment:
NIT: I wonder if we should check for `button.visible` instead here, since that's ultimately what we're really after
```
const visibleRightContainerButtons = this.rightContainer.children.filter(button => button.visible);
```
i think we dont do that above because `dropped` is only checked inside `layout`, which hasn't been called yet

Reply:
Excellent point, I'll do this in the commit.